runtime/fastly/builtins/device.h

```diff
@@ -0,0 +1,62 @@
+#ifndef FASTLY_DEVICE_H
+#define FASTLY_DEVICE_H
+
+#include "builtin.h"
+#include "extension-api.h"
+
+using builtins::BuiltinImpl;
```

Comment:
It's good to keep `using` declarations out of headers, as they change the namespace everywhere they're included. Could you move this into the cpp file, and use the fully qualified name in the header?

Reply:
Sure, added.